src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimantResponseAgreedSettledPartAdmitDefendantLipNotificationHandlerTest.java

```diff
@@ -49,6 +51,7 @@ class AboutToSubmitCallback {
         @BeforeEach
         void setup() {
             when(notificationsProperties.getRespondentLipPartAdmitSettleClaimTemplate()).thenReturn("spec-lip-template-id");
+            when(notificationsProperties.getRespondentLipPartAdmitSettleClaimBilingualTemplate()).thenReturn("spec-lip-template-id");
```

Comment:
Are you using the same template id in the test for english and bilingual? Better use a different template id, otherwise the test will not test the task.

Reply:
Updated